django_declarative_apis/machinery/filtering.py

```diff
@@ -107,22 +146,60 @@ def _get_filtered_field_value(
         return None
 
 
+def _make_filter_cache_key(expand_children, klass, pk):
+    return (str(expand_children), klass, str(pk))
```

Comment:
`klass` would end up being the string repr of the instance's class, right? Would `f"{klass.__module__}.{klass.__name__}"` be slightly preferable?

Reply:
`klass` is the actual `type` object, not a str.

`expand_children` has to be stringified because `dicts` are not hashable.  `pk` has to be stringified because otherwise we get a mix of `uuid` and `str` that don't match like they should.

Reply:
Ah, I'm dumb. I was thinking it was being serialized for some reason.